thebaristas/maritime-for-a-latte
# support mobile

To test the changes on iOS/Android:
- Unity > Build Settings > iOS/Android
- Press play in Unity
- Sell button should appear and tutorial text should change
